toolbox: add pages (#8280)
tldr-pages/tldr: pages/linux/toolbox-rm.md

# toolbox rm

> Remove one or more toolbox conatiners.
> More information: <https://www.mankier.com/1/toolbox-rm>.

- Remove all `toolbox` containers:

`toolbox rm --all`

> - Force the removal of running and paused `toolbox` containers:

Author: ![image](https://user-images.githubusercontent.com/26346867/182395546-9effac14-3d6b-454f-baa2-4634de24db50.png)
Running containers/toolboxes are the ones I am currently working on or logged into. As you can see in the image above there is a container that I am currently logged into and the other one is paused.

> - Force the removal of running and paused `toolbox` containers:

Reviewer: Then how about
```suggestion
- Force the removal of all currently active `toolbox` containers:
```

> - Force the removal of running and paused `toolbox` containers:

Author: Looks good

# toolbox rm

> Remove one or more `toolbox` containers.
> More information: <https://manned.org/toolbox-rm.1>.

- Remove a toolbox container:

> `toolbox rm {{container_name}}`

Reviewer: Where does this container name come from?

- Remove all `toolbox` containers:

`toolbox rm --all`

> - Force the removal of all currently active `toolbox` containers:
> 
> `toolbox rm --force`

Reviewer: From the man page it sounds more like the command should look like this:
```suggestion
- Force the removal of a currently active `toolbox` container:

`toolbox rm --force {{container_name}}`
```
Is this correct?

Author: Yes, It is correct
